packages/rollup/src/executors/rollup/rollup.impl.ts

```diff
@@ -306,7 +306,7 @@ function createTsCompilerOptions(
 ) {
   const compilerOptionPaths = computeCompilerOptionsPaths(config, dependencies);
   const compilerOptions = {
-    rootDir: options.projectRoot,
+    rootDir: config.options.rootDir,
```

Comment:
`options.projectRoot` contains path to root of particular package. `config.options.rootDir` contain path to workspace root.